app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -229,5 +216,34 @@ data class Patient(
         yesAnswer: MedicalHistory.Answer = MedicalHistory.Answer.YES,
         scheduled: Appointment.Status = Appointment.Status.SCHEDULED
     ): Flowable<Boolean>
+
+    companion object {
+      @Language("RoomSql")
+      const val patientProfileQuery = """
+        SELECT
+          P.uuid patient_uuid, P.addressUuid patient_addressUuid, P.fullName patient_fullName,
+          P.searchableName patient_searchableName, P.gender patient_gender, P.dateOfBirth patient_dateOfBirth,
+          P.age_value patient_age_value, P.age_updatedAt patient_age_updatedAt,
+          P.age_computedDateOfBirth patient_age_computedDateOfBirth, P.status patient_status,
+          P.createdAt patient_createdAt, P.updatedAt patient_updatedAt, P.deletedAt patient_deletedAt,
+          P.syncStatus patient_syncStatus,
+
+          PA.uuid addr_uuid, PA.colonyOrVillage addr_colonyOrVillage, PA.district addr_district,
+          PA.state addr_state, PA.country addr_country,
+          PA.createdAt addr_createdAt,PA.updatedAt addr_updatedAt, PA.deletedAt addr_deletedAt,
+
+          PPN.uuid phone_uuid, PPN.patientUuid phone_patientUuid, PPN.number phone_number,
+          PPN.phoneType phone_phoneType, PPN.active phone_active,
+          PPN.createdAt phone_createdAt, PPN.updatedAt phone_updatedAt, PPN.deletedAt phone_deletedAt,
+
+          BI.uuid businessid_uuid, BI.patientUuid businessid_patientUuid, BI.identifier businessid_identifier,
+          BI.identifierType businessid_identifierType, BI.meta businessid_meta, BI.metaVersion businessid_metaVersion,
+          BI.createdAt businessid_createdAt, BI.updatedAt businessid_updatedAt, BI.deletedAt businessid_deletedAt
+        FROM Patient P
+        INNER JOIN PatientAddress PA ON P.addressUuid == PA.uuid
```

Comment:
Is there a difference between single equals vs double equals? We've used single equals everywhere else.

Reply:
In SQLite, both are acceptable and they are the same thing. I think @sanchita looked this up recently.

Reply:
Both are essentially the same in SQLite.